src/common/__tests__/BaseAnnotator-test.ts

```diff
@@ -195,6 +195,14 @@ describe('BaseAnnotator', () => {
             expect(annotator.removeListener).toBeCalledWith(Event.VISIBLE_SET, expect.any(Function));
             expect(annotator.removeListener).toBeCalledWith(LegacyEvent.SCALE, expect.any(Function));
         });
+
+        test('should remove mousedown event listener', () => {
+            const removeEventListenerSpy = jest.spyOn(document, 'removeEventListener');
+
+            annotator.destroy();
+
+            expect(removeEventListenerSpy).toHaveBeenCalledWith('mousedown', expect.any(Function));
```

Comment:
Since its visibility is `protected` typescript complains about attempting to reference it. It appears the other event callbacks follow this `expect.any(Function)` approach as well. I tried looking for a disable rule for the tslint but haven't been able to find one